8-web-components/4-template-element/article.md

```diff
@@ -1,13 +1,14 @@
 
 # Template element
```

Comment:
Also please translate the title somehow so it'll be obvious in the site that it's been translated.